selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/ImportWizardFormTest.java

```diff
@@ -0,0 +1,614 @@
+        Paths.get(getClass().getResource("/projects/testRepo").getPath()), SECOND_SUBMODULE_NAME);
+  }
+
+  private void loginToGitHub() {
+    try {
+      askDialog.waitFormToOpen(25);
+    } catch (TimeoutException ex) {
+      importProject.closeWithIcon();
+      events.clickEventLogBtn();
+      events.waitExpectedMessage("Failed to authorize application on GitHub");
+      fail("Known issue https://github.com/eclipse/che/issues/8288", ex);
+    }
+  }
+
+  private void importRecursivelyFromGitHub(String projectName) throws Exception {
+    loader.waitOnClosed();
+    menu.runCommand(
+        TestMenuCommandsConstants.Workspace.WORKSPACE,
+        TestMenuCommandsConstants.Workspace.IMPORT_PROJECT);
+    importProject.waitMainForm();
+    loader.waitOnClosed();
+    importProject.selectGitHubSourceItem();
+    loader.waitOnClosed();
+    importProject.waitLoadRepoBtn();
+    importProject.clickLoadRepoBtn();
+    loader.waitOnClosed();
+    importProject.selectItemInAccountList(
+        gitHubClientService.getName(gitHubUsername, gitHubPassword));
+    importProject.selectProjectByName(projectName);
+    importProject.typeProjectName(projectName);
+    importProject.waitImportRecursivelyIsNotSelected();
+    importProject.clickOnImportRecursivelyCheckbox();
+    importProject.waitImportRecursivelyIsSelected();
+    importProject.clickImportBtn();
+    importProject.waitMainFormIsClosed();
+    loader.waitOnClosed();
+    projectWizard.waitCreateProjectWizardForm();
+    projectWizard.selectTypeProject(Wizard.TypeProject.MAVEN);
+    loader.waitOnClosed();
+    projectWizard.clickSaveButton();
+    loader.waitOnClosed();
+    projectWizard.waitCreateProjectWizardFormIsClosed();
+    projectExplorer.waitProjectExplorer();
+    projectExplorer.waitItem(projectName);
+  }
+
+  private void openSubmoduleOne(String projectName) throws Exception {
+    projectExplorer.openItemByPath(projectName);
+    projectExplorer.waitAndSelectItem(projectName + "/" + FIRST_SUBMODULE_NAME);
+    menu.runCommand(
+        TestMenuCommandsConstants.Project.PROJECT,
+        TestMenuCommandsConstants.Project.CONVERT_TO_PROJECT);
+    projectWizard.waitOpenProjectConfigForm();
+    projectWizard.waitTextParentDirectoryName("/" + projectName);
+    projectWizard.waitTextProjectNameInput(FIRST_SUBMODULE_NAME);
+    projectWizard.selectTypeProject(Wizard.TypeProject.MAVEN);
+    loader.waitOnClosed();
+    projectWizard.clickSaveButton();
+    projectWizard.waitCloseProjectConfigForm();
+    loader.waitOnClosed();
+    projectExplorer.waitItem(projectName);
+    projectExplorer.expandPathInProjectExplorerAndOpenFile(
+        projectName + "/" + FIRST_SUBMODULE_NAME + "/src/main/java/com.codenvy.example.spring",
+        "GreetingController.java");
+    projectExplorer.waitVisibilityByName("HelloWorld.java");
+    editor.closeFileByNameWithSaving("GreetingController");
+  }
+
+  private void openSubmoduleTwo(String projectName) throws Exception {
```

Comment:
It's better not to enclose submodule number into the method name because it makes it hard to reorder the modules in future.